scripts/pwa-shell.test.ts

```
@@ -0,0 +1,94 @@
+import { createHash } from 'node:crypto';
+import assert from 'node:assert/strict';
```

ESLint

[error] `node:assert/strict` import should occur before import of `node:crypto`

(import/order)